octodns_azure/__init__.py

```diff
@@ -285,33 +285,33 @@ def _pool_traffic_manager_name(pool, record):
     return f'{prefix}-pool-{pool}'
 
 
-def _healthcheck_tolerated_number_of_failures(record):
+def _healthcheck_num_failures(record):
     return record._octodns.get('azuredns', {}) \
         .get('healthcheck', {}) \
-        .get('tolerated_number_of_failures')
+        .get('num_failures', 3)
```

Comment:
Any concerns around people already using these values? Also looks like they'll now get a default rather than error if they were using the old one.

Reply:
We haven't made a release since this feature was added, so I'm ok changing it. I also don't foresee anyone using it yet so I'm ok with this change.